docs/details/local_search.rst

```diff
@@ -0,0 +1,30 @@
+==========================================================
+Searching numerical values for parameters via local search
+==========================================================
+
+hal-cgp supports the use of numerical values in expression via nodes with input-independent output in order to represent expressions like `f(x) = x + c` with some fixed value `c`.
+
+While the `ConstantFloat` node has a fixed output value, the output value of a `Parameter` node is stored per individual and can be modified during evolution via local search. These parameters can be different for each position in the genome.
```

Comment:
```suggestion
While the `ConstantFloat` node has a fixed output value, the output value of a `Parameter` node is stored per individual and can be modified during evolution via local search. These parameters can be different for each position in the genome. One can define custom nodes which contain an arbitrary number of parameters (see `example_parametrized_nodes.py`)
```
I think it would be worth making the link to the example here